apps/generator/lib/templates/config/loader.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const { readFile } = require('../../utils');
+const log = require('loglevel');
+
+const CONFIG_FILENAME = 'package.json';
```

Comment:
use the opportunity to refactro this too, so we now have 2 config files
- `PRIMARY_CONFIG_FILE` which is `.ageneratorrc`
- `SECONDARY_CONFIG_FILE` which is `package.json`